lib/components/map/default-map.js

```diff
@@ -200,6 +199,8 @@ class DefaultMap extends Component {
               default: return null
             }
           })}
+          {/* Render custom overlays, if set. */}
+          {getCustomMapOverlays && getCustomMapOverlays()}
```

Comment:
Might be worth doing a typeof check here.
```suggestion
          {typeof getCustomMapOverlays === 'function' && getCustomMapOverlays()}
```

Reply:
Yep, added in https://github.com/opentripplanner/otp-react-redux/pull/401/commits/4348cdb2c20d2658d9b4f113dd15e18d3b110939.